src/Cnab.JsonConverters/ParameterConverter.cs

```diff
@@ -44,6 +44,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
         private Parameter populateParameter(JsonReader reader, JsonSerializer serializer, Parameter destination)
         {
             serializer.Populate(reader, destination);
+            
```

Comment:
Ugh, I clearly miss `gofmt` and `golint`...

Reply:
OmniSharp or [this one](https://marketplace.visualstudio.com/items?itemName=Leopotam.csharpfixformat). I don't really trust formatters, apologies.